src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,14 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        String s = "";
```

Comment:
try to use full name of the variable

Reply:
Full name is used.  Now it's binaryString instead of s. 
Done!